### PR TITLE
add 125 valid-palindrome

### LINKED ISSUE #61: add 125 Valid Palindrome
## Summary
add 125

## Goal
add 125

## Todo
- [ ] add 125
- [ ] write time and space complexity

## Deadline
5/4

## Parent issue
None

## References
None

## Notes
None

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 125.valid-palindrome.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# @lc code=start
+class Solution:
+    def isPalindrome(self, s: str) -> bool:
+        """
+        time: O(len(s))
+        space: O(len(s))
+        """
+        only_alphabet_num_list = []
+        for word in s.lower():
```
Comment (ryuji0123) [inline 125.valid-palindrome.py:15]: word -> character or char

File: 125.valid-palindrome.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#
+
+# @lc code=start
+class Solution:
+    def isPalindrome(self, s: str) -> bool:
+        """
+        time: O(len(s))
+        space: O(len(s))
+        """
+        only_alphabet_num_list = []
```
Comment (ryuji0123) [inline 125.valid-palindrome.py:14]: use filter function to define only_alphabet_num_list

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I changed variable name and used filter function.  Please check my code.

**Pull request closed without merging**